bokeh/models/selections.py

```diff
@@ -87,7 +89,9 @@ def __init__(self, *args, **kwargs) -> None:
     keys of this dictionary (converted to ints).
     """)
 
-    # TODO (bev) image_indicies
+    image_indices = List(Struct(index=Int, i=Int, j=Int, flat_index=Int), help="""
```

Comment:
Any better options here? This works and returns e.g. 
```
[{'index': 2, 'i': 12, 'j': 11, 'flat_index': 232}]
```
but it would be lovely if a `TypedDict` or even dataclass could be resolved. I don't want to use a `DataModel` or new Model because I don't want the sub-properties to be "addressable" as properties.

Reply:
Perhaps the Python side `Struct` property could be configured with a dataclass to init for the value? I am not sure how far down the property system changes would need to go.